src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import {useEffect} from 'react'
+import ArticleCard from './ArticleCard'
+
+function ArticleList ({articles, setArticles}) {
+    useEffect(() => {
+        const getArticles = () => {
+            return axios.get("https://nc-news-lo7q.onrender.com/api/articles")
+                .then(({ data }) => {
+                    setArticles(data.articles)
+                })
+        }
+        getArticles()
```

Comment:
really good to see _Axios_ being used! As a general tip you could have your `getArticles` function in it's own file and import it into this component, that way you can also import it into other components if you need to use it elsewhere, as well as generally looking tidier 👍

Reply:
just a note here that if you do move the API call function to it's own file, you will still need to set the state `setArticles` in the component useEffect, the API call only needs to return the data